pallets/automation-time/src/migrations/update_xcmp_task.rs

```diff
@@ -189,16 +148,26 @@ mod test {
 				},
 			};
 
-			super::AccountTasks::<Test>::insert(account_id.clone(), task_id, task);
+			let old_task_id =
+				generate_old_task_id::<Test>(account_id.clone(), task.provided_id.clone());
+			super::AccountTasks::<Test>::insert(account_id.clone(), old_task_id.clone(), task);
 
 			UpdateXcmpTask::<Test>::on_runtime_upgrade();
 
 			assert_eq!(crate::AccountTasks::<Test>::iter().count(), 1);
+
+			// 0xd1263842e34adeb00be1146b30bc6527951f0a0f5d5a3f7a758537735b8bcb04
```

Comment:
Please search "0xd1263842e34adeb00be1146b30bc6527951f0a0f5d5a3f7a758537735b8bcb04" and replace with string literal.